test/noop-set.test.js

```diff
@@ -1,17 +1,17 @@
 'use strict'
 
-const tap = require('tap')
+const { test } = require('node:test')
 const noopSet = require('../lib/noop-set')
 
-tap.test('does a lot of nothing', async t => {
+test('does a lot of nothing', async t => {
   const aSet = noopSet()
-  t.type(aSet, 'object')
+  t.assert.ok(aSet, 'object')
 
   const item = {}
   aSet.add(item)
   aSet.add({ another: 'item' })
   aSet.delete(item)
-  t.equal(aSet.has(item), true)
+  t.assert.equal(aSet.has(item), true)
```

Comment:
nit: can we use `strictEqual` here
```suggestion
  t.assert.strictEqual(aSet.has(item), true)
```